samples/contrib/versioned-pipeline-ci-samples/kaggle-ci-sample/create_pipeline_version_and_run.py

```diff
@@ -0,0 +1,54 @@
+import kfp
+import argparse
+
+parser = argparse.ArgumentParser()
+parser.add_argument('--version_name', help='Required. Name of the new version. Must be unique.', type=str)
+parser.add_argument('--package_url', help='Required. pipeline package url', type=str)
+parser.add_argument('--pipeline_id', help = 'Required. pipeline id',type=str)
+parser.add_argument('--bucket_name', help='Required. gs bucket to store files', type=str)
+parser.add_argument('--gcr_address', help='Required. Cloud registry address. For example, gcr.io/my-project', type=str)
+parser.add_argument('--host', help='Host address of kfp.Client. Will be get from cluster automatically', type=str, default='')
+parser.add_argument('--run_name', help='name of the new run.', type=str, default='')
+parser.add_argument('--experiment_id', help = 'experiment id',type=str)
+parser.add_argument('--code_source_url', help = 'url of source code', type=str, default='')
+args = parser.parse_args()
+
+if args.host:
+    client = kfp.Client(host=args.host)
+else:
+    client = kfp.Client()
+
+print('your client configuration is :{}'.format(client.pipelines.api_client.configuration.__dict__))
+print('Now in create_pipeline_version_and_run.py...')
+print('your api_client host is:')
```

Comment:
nit: maybe also use `.format()` as above?

Reply:
Will modify this part.